src/app/pages/game-list/lobby/lobby.component.ts

```diff
@@ -20,7 +20,7 @@ export class LobbyComponent implements OnInit {
 
   ngOnInit() {}
 
-  getNonHostPlayers(): Player[] {
+  getNonHostPlayers(): User[] {
```

Comment:
I think this one should still be Player, not User. Since all we were using was the name attribute, it shouldn't break anything else to turn this to Player.